src/examples/footer/simple.html

```diff
@@ -0,0 +1,32 @@
+<footer class="text-base-content ">
+    <div class="footer footer-center p-10 bg-base-200">
+
+        <!-- Socials -->
+        <nav class="grid grid-flow-col gap-6">
+            <a class="fa-brands fa-github text-2xl"></a>
+            <a class="fa-brands fa-twitter text-2xl"></a>
+            <a class="fa-brands fa-facebook text-2xl"></a>
+            <a class="fa-brands fa-youtube text-2xl"></a>
```

Comment:
Add `hover:cursor-pointer` or `btn` so users can see that these icons are clickable